src/mailer.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const api_key = process.env.MAILGUN_SECRET;
+const domain = process.env.MAILGUN_DOMAIN;
+const mailgun = require('mailgun-js')({apiKey: api_key, domain: domain});
```

Comment:
If you named your `api_key` constant `apiKey`, you could pass the configuration like this:

```javascript
const mailgun = require('mailgun-js')({ apiKey, domain });
```

This is because, in ES6, if you create an object using the `{ ... }` shorthand and you have a variable of the same name as the desired object property, you no longer need to do `{ foo: foo, bar: bar }`.

More info: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer